src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,16 +105,28 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        String result = someDate.toString();
+        LocalDate now = LocalDate.now();
+        return someDate.compareTo(now) > 0
+                ? result + " is after " + now
+                : someDate.compareTo(now) < 0
+                ? result + " is before " + now
+                : result + " is today";
```

Comment:
let's use isAfter() and isBefore() , those methods are more relevant in this case